jexl-parser/src/parser.lalrpop

```diff
@@ -116,6 +117,10 @@ Op50: OpCode = {
     "^" => OpCode::Exponent,
 };
 
+Op60: UnOpCode = {
```

Comment:
Nit: given this is at `Expr70` level, to follow the rest of the code, could we name this `Op80`?